velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -203,6 +206,11 @@ std::optional<std::string> PrestoQueryRunner::toSql(
     return toSql(valuesNode);
   }
 
+  if (const auto tableScanNode =
+          std::dynamic_pointer_cast<const core::TableScanNode>(plan)) {
```

Comment:
When do we call toSql() on TableScan nodes? I remember we used to only convert the basic query plan that reads from Values node to SQL.

Reply:
I saw PrestoQueryRunnerTest fail after I enabled toSql on the source of an AggregationNode without this.

If it's not used anywhere else I can update the tests to use values.

Reply:
@kevinwilfong Thank you for the pointer. I think it's OK to add this toSql(tableScan). Though, shall we make it return the table name from HiveTableConnector in the TableScanNode instead a hard-coded "tmp"? (We can throw NYI if the TableConnector is not a HiveTableConnector.)

Reply:
@kevinwilfong By the way, I saw JoinFuzzer failing on this PR: https://github.com/facebookincubator/velox/actions/runs/12383032238/job/34566476030?pr=11897. Please also take a look.